### PR TITLE
Removing Thread stack use

### PR DESCRIPTION
**Pull request by iriark01**
This is a little hard because Thread is also an RTOS thing. 

Please note: we have a thread example that I don't know how to change: mesh_tutorial, line 106 (or 97 in the new version).

### CODE REVIEW

File: docs/reference/configuration/Connectivity.md [hunk 1]
```diff
@@ -23,11 +23,6 @@ Name: lwip.debug-enabled
     Description: Enable debug trace support
     Defined by: library:lwip
     No value set
-Name: lwip.default-thread-stacksize
```
Comment (mikter) [inline docs/reference/configuration/Connectivity.md]: This is not Mesh Thread stuff don't delete

File: docs/reference/configuration/Connectivity.md [hunk 2]
```diff
@@ -81,11 +76,6 @@ Name: lwip.ppp-ipv6-enabled
     Description: Enable support for ipv6 PPP interface
     Defined by: library:lwip
     No value set
-Name: lwip.ppp-thread-stacksize
```
Comment (mikter) [inline docs/reference/configuration/Connectivity.md]: This is not Mesh Thread stuff don't delete

File: docs/reference/configuration/Connectivity.md [hunk 3]
```diff
@@ -127,11 +117,6 @@ Name: lwip.tcp-wnd
     Description: TCP sender buffer space (bytes). Current default (used if null here) is set to (4 * TCP_MSS) in opt.h, unless overridden by target Ethernet drivers.
     Defined by: library:lwip
     No value set
-Name: lwip.tcpip-thread-stacksize
```
Comment (mikter) [inline docs/reference/configuration/Connectivity.md]: This is not Mesh Thread stuff don't delete

File: docs/tutorials/using_apis/connectivity/mesh_tutorial.md [hunk 4]
```diff
@@ -98,9 +88,10 @@ Connect to the mesh network:
     }
 ```
 
-The application configuration file `mbed_app.json` defines the mesh network type (Wi-SUN, Thread or 6LoWPAN-ND), RF radio shield and other configuration parameters.
+The application configuration file `mbed_app.json` defines the mesh network type (Wi-SUN or 6LoWPAN-ND), RF radio shield and other configuration parameters.
 
 Below is an example configuration file for a Thread device connected to an Atmel RF AT86RF233 shield:
```
Comment (iriark01) [inline docs/tutorials/using_apis/connectivity/mesh_tutorial.md]: If this supports non-thread, how do I phrase it? Or is it a completely different topic?

**Comment by iriark01** [comment]
@mikter please can you approve/reject?

File: docs/tutorials/using_apis/connectivity/mesh_tutorial.md (hunk 4 above)
Comment (mikter) [reply docs/tutorials/using_apis/connectivity/mesh_tutorial.md]: Replace Thread with Mesh device we can run any mesh on top any radio so it is ok

**Review by mikter — APPROVED** [review]

File: docs/tutorials/using_apis/connectivity/mesh_tutorial.md (hunk 4 above)
Comment (iriark01) [reply docs/tutorials/using_apis/connectivity/mesh_tutorial.md]: See commit: https://github.com/ARMmbed/mbed-os-5-docs/pull/1310/commits/90a0d57aa020807d6aff4a7a16be288f0787111e

**Review by mikter — APPROVED** [review]

**Pull request merged**